galaxy_ng/app/constants.py

```diff
@@ -16,3 +16,290 @@ class DeploymentMode(enum.Enum):
 )
 
 INBOUND_REPO_NAME_FORMAT = "inbound-{namespace_name}"
+
+PERMISSIONS = {
+    "galaxy.add_namespace": {
+        # Short name to display in the UI
+        "name": _("Add namespace"),
+
+        # Description to use when the permission is being assigned to a specifc object
+        "object_description": None,
+
+        # Description to use when the permission is being assigned globally
+        "global_description": "Create a new namespace.",
+
+        # Category to group the permission in the UI.
+        "ui_category": _("Collection Namespaces"),
+
+    },
+    "galaxy.change_namespace": {
+        "name": _("Change namespace"),
+
+        "object_description": "Edit this namespace.",
```

Comment:
note all of these have to be translated (including object_description and global_description), not only name and ui_category